### ISSUE TITLE
Slider and Pivot not working correctly in Edge under Windows High-contrast Black mode

### ISSUE THREAD

**Issue by marksoper** [OfficeDev/office-ui-fabric-react#12208]
We believe we have found two Office Fabric bugs that are showing up in AzureML Studio in this bug:
https://msdata.visualstudio.com/Vienna/_workitems/edit/537092

To recreate, use these links:

1) https://developer.microsoft.com/en-us/fabric#/controls/web/slider
Slider track disappears as the slider moves to the right

2) https://developer.microsoft.com/en-us/fabric#/controls/web/pivot
Pivot Button text not visible when active

**Comment by aneeshack4** [comment]
Hi @marksoper, thanks for reaching out to us. I see the Pivot repro:
![image](https://user-images.githubusercontent.com/4161791/76041401-d66afe00-5f06-11ea-8c54-a9de28a9f84c.png)
but not the Slider repro:
![image](https://user-images.githubusercontent.com/4161791/76041413-df5bcf80-5f06-11ea-9f7f-4291f50541e1.png)
(the track doesn't disappear as slider moves to the right)

**Comment by beyackle** [comment]
I've run into this problem as well; when I turn on high-contrast mode, the Pivot examples in the docs fail to render correctly in Edge.
![pivot-hc](https://user-images.githubusercontent.com/61990921/76465528-46640300-63a3-11ea-9bd9-1b9e2233b3c4.png)
